compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensions.cpp

```diff
@@ -204,6 +204,10 @@ DiagnosedSilenceableFailure transform_dialect::ApplyPatternsOp::applyToOne(
   if (getSwappingPatterns())
     addSwappingPatterns(patterns, getSwapPaddingElideConditional());
   if (getAdditionalIreePatterns()) addAdditionalIreePatterns(patterns);
+  if (getBubbleCollapseExpand()) {
```

Comment:
can you split that part and send it to main so we can rebase on top?

Reply:
I did this in #11344, but I don't have the time to do the rebase. More generally, there is little point in having a feature branch if such minor changes need to be sent to main and the entire branch rebased.